pmm-app/tests/upgradePMM_test.js

```diff
@@ -4,92 +4,132 @@ const serviceNames = {
   postgresql: 'postgres_upgrade_service',
   mongodb: 'mongodb_upgrade_service',
   proxysql: 'proxysql_upgrade_service',
-  rds: 'mysql_rds_uprgade_service'
+  rds: 'mysql_rds_uprgade_service',
 };
 
 Feature('PMM server Upgrade');
 
-Before(async (I) => {
+Before(async I => {
   I.Authorize();
   I.setRequestTimeout(30000);
 });
 
 Scenario(
-    'Verify user can create Remote Instances before upgrade and they are in RUNNNING status @pmm-upgrade @visual-test @not-pr-pipeline',
-    async (I, homePage, inventoryAPI, addInstanceAPI) => {
-      // Adding instances for monitoring
-      for (const type of Object.values(addInstanceAPI.instanceTypes)) {
-        await addInstanceAPI.apiAddInstance(type, serviceNames[type.toLowerCase()]);
-      }
-      // Checking that instances are RUNNING
-      for (const service of Object.values(inventoryAPI.services)) {
-        await inventoryAPI.verifyServiceExistsAndHasRunningStatus(service,
-            serviceNames[service.service]);
-      }
+  'PMM-T289 Verify Whats New link is presented on Update Widget @pmm-upgrade @visual-test @not-pr-pipeline',
+  async (I, homePage) => {
+    I.amOnPage(homePage.url);
+    const res2 = process.env.DOCKER_VERSION.split('.');
+    const res = process.env.PMM_SERVER_LATEST.split('.');
+    const majorVersionDiff = res[1] - res2[1];
+    const patchVersionDiff = res[2] - res2[2];
```

Comment:
```suggestion
    const [, pmmMajor, pmmMinor] = process.env.PMM_SERVER_LATEST.split('.');
    const [, dockerMajor, dockerMinor] = process.env.DOCKER_VERSION.split('.');
    const majorVersionDiff = pmmMajor - dockerMajor;
    const patchVersionDiff = pmmMinor - dockerMinor;
```

Reply:
1/ This would make it easier to understand what it does.
2/ We would also need to write a comment providing some details to this comparison (i.e. why)

Reply:
sure makes sense, I have tried added clarification and followed your suggestion.